backend/main.py

```
@@ -1608,5 +1683,176 @@ def send_status_notification(issue_id: int, old_status: str, new_status: str, no
     finally:
         db.close()
 
+# Escalation API Endpoints
+@app.get("/api/grievances", response_model=List[GrievanceSummaryResponse])
+def get_grievances(
+    status: Optional[str] = Query(None, description="Filter by status"),
```

Ruff (0.14.14)

[error] Undefined name `Optional`

(F821)